Car/TeamNull SmartCar/src/util/HardwareUtil.cpp

```diff
@@ -77,11 +83,16 @@ void initialiseSensors() {
   delay(150);
   rearSensor.setTimeout(500);
   // rearSensor.setAddress((uint8_t)01);
-  if (!rearSensor.init()) {
+  while (!rearSensor.init()) {
     Serial.println("Failed to initialise REAR sensor");
-    blink(10);
-    while (1) {
-    }
+    blink(2);
+    pinMode(frontSensorPin, OUTPUT);
+    digitalWrite(frontSensorPin, LOW);
+
+    delay(1500);
+    digitalWrite(frontSensorPin, HIGH);
+    delay(200);
+    pinMode(frontSensorPin, INPUT);
```

Comment:
Why are you flipping the pins and not using the Smartcar library?

Reply:
Perhaps ignorance. I am not aware that the library supports the VL53L0X sensors directly, and further did not find a section in the documentation referring to the setting of pins? So I'm not entirely sure how we should be implementing the library here? Could you please point us to a section in the documentation, or otherwise guide us, so that this can be resolved?

Reply:
At first I thought you were trying to use an ultrasonic sensor but that's not the case. I have no idea what you are trying to do with your code. Left some additional comments to be more exact.